internal/activity/activity.go

```diff
@@ -56,6 +58,7 @@ func TriggerHTTP(ctx context.Context, orgID, userID string, trigger TriggerMetho
 		ai.Path,
 		ai.RequestMethod,
 		"",
+		"",
```

Comment:
why is there no http status in the triggerHTTP?

Reply:
Currently the "pure" HTTP calls are OIDC and SAML calls, which are in the storage logic provided by ZITADEL and used in the respective libraries, problem there is that as the calls are not yet finished there is no HTTP status.
I did some changes which could provide us with the necessary info in OIDC like this https://github.com/zitadel/oidc/pull/490